zebra-chain/src/transparent/script.rs

```diff
@@ -48,6 +56,26 @@ impl fmt::Debug for Script {
     }
 }
 
+impl ToHex for &Script {
+    fn encode_hex<T: FromIterator<char>>(&self) -> T {
+        self.as_raw_bytes().encode_hex()
```

Comment:
Other implementations of this (like `Hash`) use a `bytes_in_display_order` method. I am not sure if this is needed for Scripts.

Reply:
It isn't, scripts are stored and displayed in the same order.